From c2010fb61826c45bdd0ee6f36c17d7bf84d8daaf Mon Sep 17 00:00:00 2001
From: Ghislain Antony Vaillant <ghisvail@gmail.com>
Date: Fri, 23 Dec 2016 16:31:56 +0000
Subject: Use system include path

---
 pybind11/__init__.py |  8 +-------
 setup.py             | 19 -------------------
 2 files changed, 1 insertion(+), 26 deletions(-)

diff --git a/pybind11/__init__.py b/pybind11/__init__.py
index a765692..a7d57c3 100644
--- a/pybind11/__init__.py
+++ b/pybind11/__init__.py
@@ -2,10 +2,4 @@ from ._version import version_info, __version__  # noqa: F401 imported but unuse
 
 
 def get_include(*args, **kwargs):
-    import os
-    try:
-        from pip import locations
-        return os.path.dirname(
-            locations.distutils_scheme('pybind11', *args, **kwargs)['headers'])
-    except ImportError:
-        return 'include'
+    return '/usr/include'
diff --git a/setup.py b/setup.py
index a25f1af..29eeb34 100644
--- a/setup.py
+++ b/setup.py
@@ -15,25 +15,6 @@ setup(
     download_url='https://github.com/wjakob/pybind11/tarball/v' + __version__,
     packages=['pybind11'],
     license='BSD',
-    headers=[
-        'include/pybind11/attr.h',
-        'include/pybind11/cast.h',
-        'include/pybind11/chrono.h',
-        'include/pybind11/common.h',
-        'include/pybind11/complex.h',
-        'include/pybind11/descr.h',
-        'include/pybind11/eigen.h',
-        'include/pybind11/eval.h',
-        'include/pybind11/functional.h',
-        'include/pybind11/numpy.h',
-        'include/pybind11/operators.h',
-        'include/pybind11/options.h',
-        'include/pybind11/pybind11.h',
-        'include/pybind11/pytypes.h',
-        'include/pybind11/stl.h',
-        'include/pybind11/stl_bind.h',
-        'include/pybind11/typeid.h'
-    ],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
