Index: pybind11/tests/CMakeLists.txt
===================================================================
--- pybind11.orig/tests/CMakeLists.txt	2020-11-06 17:15:05.331222406 +0800
+++ pybind11/tests/CMakeLists.txt	2020-11-06 17:15:47.551512934 +0800
@@ -39,6 +39,10 @@
   endif()
 endmacro()
 
+if(DEFINED PYVERSION)
+  find_package(Python ${PYVERSION} EXACT COMPONENTS Interpreter Development)
+endif()
+
 # New Python support
 if(DEFINED Python_EXECUTABLE)
   set(PYTHON_EXECUTABLE "${Python_EXECUTABLE}")
