From: Ghislain Antony Vaillant <ghisvail@gmail.com>
Date: Fri, 23 Dec 2016 16:31:56 +0000
Subject: Use the system include path

---
 pybind11/__init__.py |  8 +-------
 setup.py             | 27 ++-------------------------
 2 files changed, 3 insertions(+), 32 deletions(-)

diff --git a/pybind11/__init__.py b/pybind11/__init__.py
index a765692..a7d57c3 100644
--- a/pybind11/__init__.py
+++ b/pybind11/__init__.py
@@ -2,10 +2,4 @@ from ._version import version_info, __version__  # noqa: F401 imported but unuse
 
 
 def get_include(*args, **kwargs):
-    import os
-    try:
-        from pip import locations
-        return os.path.dirname(
-            locations.distutils_scheme('pybind11', *args, **kwargs)['headers'])
-    except ImportError:
-        return 'include'
+    return '/usr/include'
diff --git a/setup.py b/setup.py
index 0cf4e47..de6a136 100644
--- a/setup.py
+++ b/setup.py
@@ -6,31 +6,8 @@ from setuptools import setup
 from pybind11 import __version__
 import os
 
-# Prevent installation of pybind11 headers by setting
-# PYBIND11_USE_CMAKE.
-if os.environ.get('PYBIND11_USE_CMAKE'):
-    headers = []
-else:
-    headers = [
-        'include/pybind11/attr.h',
-        'include/pybind11/cast.h',
-        'include/pybind11/chrono.h',
-        'include/pybind11/class_support.h',
-        'include/pybind11/common.h',
-        'include/pybind11/complex.h',
-        'include/pybind11/descr.h',
-        'include/pybind11/eigen.h',
-        'include/pybind11/eval.h',
-        'include/pybind11/functional.h',
-        'include/pybind11/numpy.h',
-        'include/pybind11/operators.h',
-        'include/pybind11/options.h',
-        'include/pybind11/pybind11.h',
-        'include/pybind11/pytypes.h',
-        'include/pybind11/stl.h',
-        'include/pybind11/stl_bind.h',
-        'include/pybind11/typeid.h'
-    ]
+# Prevent installation of pybind11 headers
+headers = []
 
 setup(
     name='pybind11',
